src/main/resources/assets/minecolonies/lang/manual_en_us.json

```diff
@@ -2399,55 +2399,56 @@
+  "com.minecolonies.building.fletcher.desc": "Crafts items using wool, string, and leather materials",
+  "com.minecolonies.building.dyer.desc": "Crafts and colors items using dyes",
+  "com.minecolonies.building.glassblower.desc": "Crafts and smelts items using glass materials",
+  "com.minecolonies.building.plantation.desc": "Cultivates plants such as bamboo, cactus, and sugarcane, and crafts items from these materials",
+  "com.minecolonies.building.concretemixer.desc": "Crafts concrete powder and concrete blocks",
+  "com.minecolonies.building.graveyard.desc": "Retrieves and cares for the graves of deceased citizens",
+  "com.minecolonies.building.netherworker.desc": "Gathers resources from the dangerous Nether, with necessary supplies and precautions",
+  "com.minecolonies.building.guardtower.desc": "Houses a guard to defend the colony and claims extra land for use",
+  "com.minecolonies.building.barrackstower.desc": "Houses many guards to defend the colony",
+  "com.minecolonies.building.warehouse.desc": "Stores and distributes resources throughout the colony with the help of couriers",
+  "com.minecolonies.building.deliveryman.desc": "Handles resource distribution and collection between the warehouse and other workers",
+  "com.minecolonies.building.university.desc": "Enables technology research led by skilled researchers",
+  "com.minecolonies.building.baker.desc": "Crafts various food items, including bread and cakes",
+  "com.minecolonies.building.barracks.desc": "Enables hiring a large number of guards and strengthens colony protection",
+  "com.minecolonies.building.cook.desc": "Serves food to hungry citizens",
+  "com.minecolonies.building.kitchen.desc": "Prepares and cooks a range of dishes for the colony",
+  "com.minecolonies.building.library.desc": "Provides a study place to improve the intelligence of adult citizens",
+  "com.minecolonies.building.smeltery.desc": "Processes ore to produce ingots through smelting",
+  "com.minecolonies.building.crusher.desc": "Crushes and grinds resources into finer materials",
+  "com.minecolonies.building.sifter.desc": "Sifts through soil to find valuable resources like gems and ingots",
+  "com.minecolonies.building.enchanter.desc": "Enhances books with enchantments and produces magical scrolls",
+  "com.minecolonies.building.composter.desc": "Composts organic materials into compost or dirt",
+  "com.minecolonies.building.alchemist.desc": "Brews potions and cultivates netherwart and mistletoe",
+  "com.minecolonies.building.simplequarry.desc": "Small excavation area for a quarrier, used for mining and extracting stones",
+  "com.minecolonies.building.mediumquarry.desc": "Large excavation area for a quarrier, used for mining and extracting stones",
   "com.minecolonies.coremod.structures.nocustomhuts": "The builder never heard of this blueprint for this building. Are you sure you handled this correctly?",
```

Comment:
The builder has never heard of this blueprint for this building. Are you sure you handled this correctly?